pkg/apis/integreatly/v1alpha1/zz_generated.openapi.go

```diff
@@ -442,19 +442,19 @@ func schema_pkg_apis_integreatly_v1alpha1_RedisSnapshot(ref common.ReferenceCall
 					},
 					"spec": {
 						SchemaProps: spec.SchemaProps{
-							Ref: ref("github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1.RedisSnapshotSpec"),
+							Ref: ref("./pkg/apis/integreatly/v1alpha1.RedisSnapshotSpec"),
```

Comment:
What's the reasoning behind this change, is it just due to a different Operator SDK version?